sklearn/neighbors/_nearest_centroid.py

```diff
@@ -79,6 +84,16 @@ class NearestCentroid(ClassifierMixin, BaseEstimator):
 
         .. versionadded:: 1.0
 
+    priors_ : array-like of shape (n_classes,)
+        Class priors (sum to 1).
+
+        .. versionadded:: 1.4
+
+    s_ : array-like of shape (n_features,)
```

Comment:
`s_` is a bad variable name. We should be able to do better.

I think that in `LinearDiscriminantAnalysis`, we call this `covariance_.`